components/ml/org.wso2.carbon.ml.commons/src/main/java/org/wso2/carbon/ml/commons/constants/MLConstants.java

```diff
@@ -170,6 +170,14 @@ private MLConstants() {
     public static final String BLOCKS = "Blocks";
     public static final String ALPHA = "Alpha";
     public static final String WEIGHTS = "Weights";
+    public static final String FOLDS = "Folds";
+    public static final String NAME_BASE_ALGORITHM = "Name_of_Base_Algorithm";
```

Comment:
It's better to rename the variable as "BASE_ALGORITHM_NAME" and the value as "Base_Algorithm_Name". Similarly for the other variables too..